fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/FilteredAggregationMaker.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+package com.yahoo.bard.webservice.data.config.metric.makers;
+
+import com.yahoo.bard.webservice.data.metric.LogicalMetric;
+import com.yahoo.bard.webservice.data.metric.MetricDictionary;
+import com.yahoo.bard.webservice.data.metric.TemplateDruidQuery;
+import com.yahoo.bard.webservice.data.metric.mappers.NoOpResultSetMapper;
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation;
+import com.yahoo.bard.webservice.druid.model.aggregation.FilteredAggregation;
+import com.yahoo.bard.webservice.druid.model.filter.Filter;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Build a Filtered Aggregation logical metric.
+ */
+public class FilteredAggregationMaker extends MetricMaker {
+
+    protected final Filter filter;
+    protected final MetricDictionary metricDictionary;
+    protected final Aggregation aggregation;
+
+    /**
+     * Construct a metric maker for Filtered Aggregations.
+     *
+     * @param metricDictionary the metric dictionary
+     * @param aggregation the underlying aggregation
+     * @param filter the filter to filter the aggregation by
+     */
+    public FilteredAggregationMaker(MetricDictionary metricDictionary, Aggregation aggregation, Filter filter) {
+        super(metricDictionary);
+        this.aggregation = aggregation;
+        this.filter = filter;
```

Comment:
Perhaps it would be better to pass in `ApiFilter` objects instead of Druid's filter model? The idea is that defining metrics is a logical operation, not a physical operation tied to Druid. Since the `FilteredAggregation` ultimately needs a Druid filter, however, this means that you'd also need to pass in a `DruidFilterBuilder` instance.

If it's the case that the API filters are not rich enough, passing in a `Collection<ApiFilter>` would likely also work as well.

Note: I'm also OK coming back to this when we re-work this maker, as long as this is mentioned in that ticket.